app/views/report/index.html.erb

```diff
@@ -1,5 +1,6 @@
 <h1>OpenFlight Carbon Leaderboard</h1>
 <h3 class="light-text">Servers with the lowest CO<sub>2</sub> emissions</h3>
+<h4 class="light-text"><%= "Grouped by identical specs" if @grouped %><h4>
```

Comment:
I think what we call the leaderboard may be different when we remove the individual leaderboard (and don't have to distinguish them) but I think this works for now